src/SWP/Bundle/FixturesBundle/Resources/themes/DefaultTheme/amp/amp-theme/partials/metadata.html.twig

```diff
@@ -0,0 +1,31 @@
+<script type="application/ld+json">
+      {
+        "@context": "http://schema.org",
+        "@type": "NewsArticle",
+        "mainEntityOfPage": "{{ url(gimme.article) }}",
+        "headline": "{{ object.title }}",
+        "datePublished": "{{ object.publishedAt|date('Y-m-d H:i:s') }}",
+        "dateModified": "{{ object.updatedAt|date('Y-m-d H:i:s') }}",
+        "description": "{{ object.lead }}",
+        "author": {
+          "@type": "Person",
+          "name": "{{ object.metadata.byline }}"
+        },
+        "publisher": {
+          "@type": "Organization",
+          "logo": {
+            "@type": "ImageObject",
+            "url": "https://assets-cdn.github.com/images/modules/logos_page/GitHub-Mark.png",
```

Comment:
okay then it should be changed to the logo of the theme for example. Because that link was just an example. @tomaszrondio should do the same for magazine theme